marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -44,6 +44,27 @@ import Welcome.Lenses (_walletLibrary)
 import Welcome.State (handleAction, dummyState, mkInitialState) as Welcome
 import Welcome.Types (Action, State) as Welcome
 
+{-
+The Marlowe Run app consists of six main workflows:
+
+1. Generate a demo wallet (this will become redundant when we integrate with real wallets).
+2. Connect a wallet (this will change when we integrate with real wallets).
+3. Disconnect a wallet.
+4. Start a contract.
+5. Move a contract forward.
+6. Redeem payments (this is triggered automatically - but may need to be manual when we
+   integrate with real wallets).
```

Comment:
This is very valuable documentation. I would love to see each of these workflows fleshed out properly into a sequence of events with domain driven design.

Reply:
mermaid.js FTW